application/server/server.go

```diff
@@ -477,28 +477,42 @@ func textDocumentDidOpenHandler() jrpc2.Handler {
 	})
 }
 
+func isDotSnykFile(uri sglsp.DocumentURI) bool {
+	return strings.HasSuffix(string(uri), ".snyk")
+}
+
+func dotSnykFileDidChangeScan(ctx context.Context, workspaceFolderPath *workspace.Folder) {
+	go workspaceFolderPath.ScanFolder(ctx)
```

Comment:
Maybe it would be easier/better to use the scan.workspaceFolder command, as there may be additional logic that would be beneficial to be executed - haven't checked, but would be possible that e.g. Trust is handled etc.

Reply:
Done 🙌